_locales/en/messages.json

```diff
@@ -12,6 +12,15 @@
         "message": "Skip over sponsorship on YouTube videos. Report sponsors on videos you watch to save the time of others.",
         "description": "Description of the extension."
     },
+    "clearTimes": {
```

Comment:
Can you add these to the bottom so it's obvious that translations are missing stuff

Reply:
How would it be obvious that there missing stuff even if there at the bottom of the file

Reply:
Since you can look at the bottom and see that the last ones are not the same.

Reply:
Moved however there may be problems with languages not being added at the same time

Reply:
Oh, will it not default back to English?

Reply:
I think it will however some translations may not be added at the same time

Reply:
If a translation is missing one string, it defaults back to English right?

Reply:
yes